test/jest/victory-scatter/helper-methods.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint no-unused-expressions: 0 */
+import * as Helpers from "victory-scatter/lib/helper-methods";
+
+describe("victory-scatter/helper-methods", () => {
+  describe("getSize", () => {
+    beforeEach(() => {
+      jest.spyOn(Helpers, "getBubbleSize");
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    const data = [
+      { x: 1, y: 2, z: 1, t: 4 },
+      { x: 2, y: 3, z: 1, t: 2 }
+    ];
+    const datum = data[0];
+    it("returns a size attribute from data", () => {
+      const point = { size: 3, ...datum };
+      const sizeResult = Helpers.getSize(point, {});
+      expect(Helpers.getBubbleSize).notCalled;
```

Comment:
```suggestion
      expect(Helpers.getBubbleSize).not.toHaveBeenCalled();
```
(and in a few other places too)